lib/matplotlib/path.py

```diff
@@ -206,12 +206,13 @@ def _fast_from_codes_and_verts(cls, verts, codes, internals=None):
         return pth
 
     def _update_values(self):
+        self._simplify_threshold = rcParams['path.simplify_threshold']
         self._should_simplify = (
             rcParams['path.simplify'] and
-            (len(self._vertices) >= 128 and
-             (self._codes is None or np.all(self._codes <= Path.LINETO)))
+            len(self._vertices) >= 128 and
+            (self._codes is None or np.all(self._codes <= Path.LINETO)) and
+            self._simplify_threshold > 0
```

Comment:
Good idea. I think combining the parameters makes a lot of sense. They are definitely redundant in purpose. I'll probably also suggest upping the default from 1/9 to something closer to 1/2. It turns out the simplify_threshold is squared in the simplification code, so 1/9 is actually only reducing paths that are 1/81 pixel away from being not parallel! Not sure what the original logic on that was. Perhaps original author was thinking of it in terms of square pixels difference...?